src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/health/HealthCheckComponentTest.kt

```diff
@@ -5,11 +5,13 @@ import org.springframework.http.HttpStatus
 import org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR
 import org.springframework.http.HttpStatus.OK
 import org.springframework.http.HttpStatus.SERVICE_UNAVAILABLE
+import org.springframework.test.context.ActiveProfiles
 import org.springframework.test.web.reactive.server.WebTestClient
 import uk.gov.justice.digital.hmpps.managerecallsapi.component.ComponentTestBase
 import java.time.LocalDateTime
 import java.time.format.DateTimeFormatter.ISO_DATE
 
+@ActiveProfiles("db-test-no-clam")
```

Comment:
OK except this copy-edit approach to config might get ugly with more permutations.

Reply:
Agreed, I tried to just inject a single property, but it didn't seem to work. Definitely one to keep an eye on.